src/s2e/Plugins/OSMonitors/Windows/WindowsMonitor.cpp

```diff
@@ -920,11 +920,19 @@ void WindowsMonitor::handleOpcodeInvocation(S2EExecutionState *state, uint64_t g
         } break;
 
         case UNLOAD_PROCESS: {
+            uint64_t returnCode;
+            if (!state->mem()->readMemoryConcrete(command.Process.EProcess + m_kernel.EProcessExitStatusOffset,
+                                                  &returnCode, sizeof(uint64_t))) {
+                getWarningsStream(state) << "could not read process return code\n";
+                break;
```

Comment:
I don't think this error should be fatal.

Reply:
Sure. Fixed.